servers/physics_3d/shape_3d_sw.h

```diff
@@ -35,7 +35,7 @@
 #include "servers/physics_server_3d.h"
 /*
 
-SHAPE_LINE, ///< plane:"plane"
+SHAPE_WORLD_MARGIN, ///< plane:"plane"
```

Comment:
Just one last nitpick: this file should only be changed as part of renaming the 3D version :)